docs/reference/configuration.rst

```diff
@@ -291,6 +299,22 @@ multiple paragraphs, if necessary. The long description *must not* be a copy of
 the ``description``, or include the ``description`` as the first line of the
 ``long_description``.
 
+``primary_color``
+~~~~~~~~~~~~~~~~~
+
+A hexadecimal RGB color value (e.g., ``#008577``) to use as the primary color
+for the application. On Android this will be used to color the app bar in the
+main window. This setting is only used if the platform allows color
+modification, otherwise it is ignored.
+
+``primary_color_dark``
+~~~~~~~~~~~~~~~~~~~~~~
+
+A hexadecimal RGB color value (e.g., ``#008577``) used alongside the primary
+color. On Android this is used to color the status bar at the top of the
+screen. This setting is only used if the platform allows color modification,
+otherwise it is ignored.
```

Comment:
As above
```suggestion
A hexadecimal RGB color value (e.g., ``#008577``) used alongside the primary
color. This setting is only used if the platform allows color modification,
otherwise it is ignored.
```